standard/interfaces.md

```diff
@@ -216,58 +218,153 @@ The members of an interface are the members inherited from the base interfaces a
 
-An *interface_declaration* creates a new declaration space ([§7.3](basic-concepts.md#73-declarations)), and the type parameters and *interface_member_declaration*s immediately contained by the *interface_declaration* introduce new members into this declaration space. The following rules apply to *interface_member_declaration*s:
+An abstract member `M1` may be overridden by either a derived interface or an implementing class or struct. That member `M1` can be accessed through a reference of the implementing type.
 
-- The name of a type parameter in the *variant_type_parameter_list* of an interface declaration shall differ from the names of all other type parameters in the same *variant_type_parameter_list* and shall differ from the names of all members of the interface.
-- The name of a method shall differ from the names of all properties and events declared in the same interface. In addition, the signature ([§7.6](basic-concepts.md#76-signatures-and-overloading)) of a method shall differ from the signatures of all other methods declared in the same interface, and two methods declared in the same interface shall not have signatures that differ solely by `in`, `out`, and `ref`.
-- The name of a property or event shall differ from the names of all other members declared in the same interface.
-- The signature of an indexer shall differ from the signatures of all other indexers declared in the same interface.
+A member `M2` with definition in an interface requires any overriding member to declare an explicit interface implementation (§19.6.2). That member `M2` only be accessed through an interface instance.
 
-The inherited members of an interface are specifically not part of the declaration space of the interface. Thus, an interface is allowed to declare a member with the same name or signature as an inherited member. When this occurs, the derived interface member is said to *hide* the base interface member. Hiding an inherited member is not considered an error, but it does cause a compiler to issue a warning. To suppress the warning, the declaration of the derived interface member shall include a `new` modifier to indicate that the derived member is intended to hide the base member. This topic is discussed further in [§7.7.2.3](basic-concepts.md#7723-hiding-through-inheritance).
+Some declarations, such as *constant_declaration* (§15.4) have no restrictions in interfaces.
 
-If a `new` modifier is included in a declaration that doesn’t hide an inherited member, a warning is issued to that effect. This warning is suppressed by removing the `new` modifier.
+The inherited members of an interface are specifically not part of the declaration space of the interface. The rules for inheritance in classes (§15.3.4) do not apply to interfaces. The `new` modifier can hide an interface member declared in a base interface. The rules for the `new` modifier in classes (§15.3.5) apply to interface member declarations.
```

Comment:
This should add that members declared in an interface are not inherited by classes implementing that interface.